symmer/chemistry/fermionic_ham.py

```diff
@@ -337,7 +340,7 @@ def __init__(
     def _single_amplitudes(self) -> FermionOperator:
         """ Calculate CC amplitudes for single excitations
         """
-        t1 = spatial2spin(self.cc_obj.t1)
+        t1 = spatial2spin(self.cc_obj.t1, orbspin=self.orbspin)
```

Comment:
Have to specify orbspin explicitly when n_alpha != n_beta - done using the pyscf.scf.addons.get_ghf_orspin function with is_rhf=True - unrestricted HF currently not implemented anyway so shouldn't be a problem.